frontend/src/App.tsx

```diff
@@ -1,36 +1,93 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
+import styled from 'styled-components';
 import logo from './logo.svg';
-import './App.css';
- 
-class App extends Component {
- 
-    state:any = {};
- 
-    componentDidMount() {
-        setInterval(this.hello, 250);
+import LandingPage from './views/Landing';
+import fetchHello from './api/hello';
+
+const StyledApp = styled.div`
+  text-align: center;
+  
+  .App-logo {
+    height: 40vmin;
+    pointer-events: none;
+  }
+  
+  @media (prefers-reduced-motion: no-preference) {
+    .App-logo {
+      animation: App-logo-spin infinite 20s linear;
     }
- 
-    hello = () => {
-        fetch('/hello')
-            .then(response => response.text())
-            .then(message => {
-                this.setState({message: message});
-            });
-    };
- 
-    render() {
-        return (
-            <div className="App">
-                <header className="App-header">
-                    <img src={logo} className="App-logo" alt="logo"/>
-                    <h1 className="App-title">{this.state.message}</h1>
-                </header>
-                <p className="App-intro">
-                    To get started, edit <code>src/App.js</code> and save to reload.
-                </p>
-            </div>
-        );
+  }
+  
+  .App-header {
+    background-color: #282c34;
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    font-size: calc(10px + 2vmin);
+    color: white;
+  }
+  
+  .App-link {
+    color: #61dafb;
+  }
+  
+  @keyframes App-logo-spin {
+    from {
+      transform: rotate(0deg);
     }
+    to {
+      transform: rotate(360deg);
+    }
+  }
+`;
+
+type MyProps = {};
+
+type MyState = {
+  message: String,
+};
+
+class App extends Component<MyProps, MyState> {
+
+  constructor(props: MyProps) {
+    super(props);
+
+    this.state = { message: 'Loading' };
+  }
+
+  componentDidMount() {
+    setInterval(this.hello, 250);
+  }
+
+  hello = () => {
+    fetchHello()
+      .then((message) => {
+        this.setState({ message });
```

Comment:
Right now, in the current setup, this actually holds a message within a message (see screenshot attachment). 
<img width="629" alt="Screen Shot 2020-08-23 at 5 41 07 PM" src="https://user-images.githubusercontent.com/7987279/90993009-f0a5a080-e567-11ea-8472-7e6f276fa2de.png">

This should be taking the actual message from the SampleObject JSON object, and displaying only the message to the frontend. (See the second screenshot below.)
<img width="386" alt="Screen Shot 2020-08-23 at 5 43 12 PM" src="https://user-images.githubusercontent.com/7987279/90993048-2cd90100-e568-11ea-947f-4b49c533e60d.png">

Reply:
Good catch, fixed.